File: src/main/java/MachTeacher/MachTeacher/dto/ConversationPreviewDTO.java
package MachTeacher.MachTeacher.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ConversationPreviewDTO {
    private Long id;
    private String mentorName;
    private String lastMessage;
    private String timeLabel;
    private int unread;
}

File: MachTeacher/app/src/main/java/com/example/machteacher/dto/ConversationPreviewDto.kt
// app/src/main/java/com/example/machteacher/dto/ConversationPreviewDto.kt
package com.example.machteacher.dto

data class ConversationPreviewDto(
    val id: Long,
    val mentorName: String,
    val lastMessage: String,
    val timeLabel: String,
    val unread: Int
)